src/styles/components/_footer.scss

```diff
@@ -1,42 +1,58 @@
 .default-footer {
-  background-color: map.get($colors, "white");
+  background-color: var(--background-color);
   width: 100%;
   position: relative;
   bottom: 0;
+  padding: 1.25rem; // 20px/16
 
   // Content wrapper
   &__content {
-    min-height: 20.75rem; /* 332px */
+    min-height: 20.75rem; /* 332px/16 */
     display: flex;
     flex-direction: column;
     justify-content: space-between;
     align-items: center;
-    gap: 0.5rem;
+    gap: 3.75rem; // 60px/16
     text-align: center;
-    padding: 3.75rem; /* 60px/16px */
-    font-size: map.get($font-sizes, "sm"); /* 20px/16px */
+    padding: 3.75rem 0; /* top&bottom: 60px/16px; left&right: 0 */
+    font-size: 1.125rem; /* 18px/16px */
     margin: auto;
-    line-height: 2rem; /* 17px/16px */
+    line-height: 1.5rem; /* 24px/16 */
     transition: 500ms color;
+    letter-spacing: 0.02em; // 2% in Figma
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      line-height: 1.0625rem; /* 17px/16px */
+      font-size: 1.25rem; /* 20px/16px */
+    }
   }
 
   // Flex items of content wrapper
   &__item {
+    color: var(--foreground-color);
     width: 100%;
 
-    & p {
-      margin: 0;
-      padding: 0;
+    // the first line of the footer has a smaller margin-bottom than the rest of the footer items.
+    // this compensates for that
+    &:first-child {
+      margin-bottom: -1.875rem; // 30px/16
+    }
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      &:first-child {
+        margin-bottom: -43px; // this is to compensate for the 17px line-height once at tablet size
+      }
     }
   }
 
-  &__item a {
+  &__item > a {
     text-decoration: underline;
-    color: map.get($colors, "black");
-    background: linear-gradient(to left, rgb(0 0 0 / 0%) 50%, map.get($colors, accent-teal) 50%);
+    color: var(--font-color);
+    background: linear-gradient(to left, transparent, var(--accent-color) 50%);
     background-size: 201% 100%;
```

Comment:
line 51 should be: 
`background: linear-gradient(to left, transparent 50%, var(--accent-color) 50%);`

This background needs an additional 50% so that we don't see the pink on the footer links BEFORE we actually do a hover.

Reply:
Thanks, I was wondering why that looked funny! I fixed that :)